packages/imaplet-lwt/imaplet-lwt.0.1.13/opam

```diff
@@ -0,0 +1,27 @@
+opam-version: "1.2"
+name: "imaplet-lwt"
+version: "0.1.13"
+maintainer: "Gregory Tsipenyuk <gregtsip@gmail.com>"
+authors: "Gregory Tsipenyuk <gregtsip@gmail.com>"
+license: "MIT"
+build: [
+  ["./configure" "--prefix=%{prefix}%"]
+  [make]
+]
+install: [make "install"]
+remove: [
+  ["ocamlfind" "remove" "imaplet"]
+  ["rm" "-f" "%{bin}%/imaplet*"]
```

Comment:
Alternatively, you can add an `install` file — [oasis2opam](https://github.com/ocaml/oasis2opam) will generate one for you.  If you use [oasis2opam](https://github.com/ocaml/oasis2opam), consider adding the version constraints to `_oasis`.